src/routes/controllers/users.controller.js

```diff
@@ -92,13 +90,14 @@ export async function getAuthorProfile(req, res) {
  */
 export const updateUserProfile = async (req, res) => {
   const { id } = req.user;
-  const { firstName, lastName, username, bio, image } = req.body;
+  const user = await getUserbyId(id);
+
+  const { firstName, lastName, bio, image } = req.body;
   try {
     const profileUpdate = await User.update(
       {
         firstName: firstName || user.firstName,
         lastName: lastName || user.lastName,
-        username: username || user.username,
```

Comment:
Is there a reason why username has been removed?

Reply:
@eben-k I removed `username` in this PR because there was no check to prevent using an existing username in the DB during the update which we prevented during signup